src/gui/screen_menu_sensor_info.cpp

```diff
@@ -38,5 +19,5 @@ void ScreenMenuSensorInfo::windowEvent(EventLock /*has private ctor*/, window_t
         return;
     }
 
-    Screen::SuperWindowEvent(sender, event, param);
+    SuperWindowEvent(sender, event, param);
```

Comment:
Please explain this change.

Reply:
It is the same.
Screen:: was unnecessary
It was just specification of parent name which contains that method